.github/workflows/test-integration.yml

```diff
@@ -0,0 +1,52 @@
+name: test-integration
+on:
+  pull_request:
+  push:
+#    branches: [ master, develop ]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    timeout-minutes: 5
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+      - name: Get Composer Cache Directory
+        id: composer-cache
+        run: echo "::set-output name=dir::$(composer config cache-files-dir)"
+      - uses: actions/cache@v2
+        with:
+          path: ${{ steps.composer-cache.outputs.dir }}
+          key: ${{ runner.os }}-composer-${{ hashFiles('**/composer.lock') }}
+          restore-keys: ${{ runner.os }}-composer-
+      - name: Get yarn cache directory path
+        id: yarn-cache-dir-path
+        run: echo "::set-output name=dir::$(yarn cache dir)"
+      - uses: actions/cache@v2
+        id: yarn-cache
+        with:
+          path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
+          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-yarn-
+      - name: Generate SSL key and certificate
+        run: |
+          sudo openssl req -x509 -nodes -days 365 -newkey rsa:2048 -keyout ./ci/files/tiqr.example.com.key -out ./ci/files/tiqr.example.com.crt -subj "/C=NL/ST=Netherlands/L=Amsterdam/O=TEST/CN=tiqr.example.com"
+      - name: Init environment
+        run: |
+          cd ci/docker
+          docker-compose up -d
+      - name: Install dependencies
+        run: |
+          cd ci/docker && docker-compose exec -T tiqr.php.test.vm.openconext.org bash -lc '
+          composer install
+          yarn
+          cp .env.ci .env
```

Comment:
Have you seen the contents of the `.env.ci`. The Symfony env is set to 'dev'. Can you check why this is the case? And (if possible) migrate this to run in the `test` environment.

Reply:
The tests should indeed use the test env instead over the dev env. It has now been changed to test.

Reply:
Good of you to add the cp .env command. In order to let it work without side effects, I suspect you'll need to move that command to a point before running composer install (as that will trigger Symfony scripts, which in turn need to run in the correct environment).

The same goes for the copying of the legacy parameters.

Reply:
Copying does now happen before the composer install call